flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/operator/CEPOperatorTest.java

```diff
@@ -923,6 +934,126 @@ public boolean filter(Event value) throws Exception {
 		}
 	}
 
+	@Test
+	public void testCEPOperatorComparatorProcessTime() throws Exception {
+		Event startEvent1 = new Event(42, "start", 1.0);
+		Event startEvent2 = new Event(42, "start", 2.0);
+		SubEvent middleEvent1 = new SubEvent(42, "foo1", 3.0, 10.0);
+		SubEvent middleEvent2 = new SubEvent(42, "foo2", 4.0, 10.0);
+		Event endEvent1 = new Event(42, "end", 1.0);
+
+		Event startEventK2 = new Event(43, "start", 1.0);
+
+		KeyedCEPPatternOperator<Event, Integer> operator = getKeyedCepOpearatorWithComparator(true);
+		OneInputStreamOperatorTestHarness<Event, Map<String, List<Event>>> harness = getCepTestHarness(operator);
+
+		try {
+			harness.open();
+
+			harness.setProcessingTime(0L);
+
+			harness.processElement(new StreamRecord<>(startEvent1, 1L));
```

Comment:
Could you update the timestamps to the `0L`(the corresponding processing time)? I know it does not matter in `ProcessingTime`, but I think it will be easier to read that way. What do you think?